liaisons/opcf/tech_reqs.md

```diff
@@ -0,0 +1,23 @@
+# Technical Requirements for OPC UA/WoT Collaboration
+
+## Use Cases
+See https://w3c.github.io/wot-usecases/#use-case-opc-ua-binding
+
+## Objectives
+1. Use OPC UA servers (device, node) from WoT Consumers; treat an OPC UA server as a Thing
+2. Access WoT TD metadata for OPC UA servers - integrate discovery process with OPC UA
```

Comment:
That should be made clearer.  There are two ways to do it: with vocabulary terms introduced in the context file (which is typically what a Binding Template defines, see what we have done for HTTP or MQTT for example), or we could also use "link" fields with specified rel types (which should probably be registered with IANA if we can't use an existing one).  The former approach is a little easier to manage.   But this should be made clearer in this document...

Reply:
BTW one place this is especially relevant is in data schemas.  For example, if a data schema includes some particular field that has a specific OPC UA definition, for example an event id, then it would (probably... we would of course have to discuss details like this) be good to define a value that can be used with an @type when that data schema is declared.

Reply:
We should look at what Apana has done, since it seems she has already been looking into how to manage this.  In general, we should build and test some prototypes before committing to a particular approach.

Reply:
I added some comments under "notes" related to this.